skycoin/skywire
# Improvements for the links in the app list

Did you run `make format && make check`?
The go code was not changed. `npm run lint` and `npm run build` were used.

Fixes #1097

 Changes:	
- A link for opening the VPN client UI was added to the app list.
- Now the link for opening the skychat app appears only if the app is running, as the UI does not work if the app is stopped.
![apps](https://user-images.githubusercontent.com/34079003/156904163-ca259686-0611-4d78-87bb-c085c9939a13.jpg)

How to test this PR:
Open the Skywire manager UI and go to the app list of a visor. Check the links in the list.

NOTE:
We could say that now there are 2 UIs for the VPN client, the one that is accessible with the link and the modal window that
appears when clicking on the gear button. @jdknives Maybe the modal window can be removed.
